### PR TITLE
Feat/add swagger

### PR DESCRIPTION
**Pull request by shaynisanov**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -18,6 +20,22 @@ app.use('/posts', postRouter);
 app.use('/comments', commentRouter);
 app.use('/users', userRouter);
 
+const options = {
```
Comment (danielpasker) [inline src/server.ts:23]: I would move it to a swagger config file

File: src/routes/commentsRoutes.ts [hunk 2]
```diff
@@ -4,18 +4,201 @@ import {authMiddleware} from '../controllers/usersController';
...
+ *         - userId
+ *         - content
+ *       properties:
+ *         _id:
+ *           type: string
+ *           description: The auto-generated id of the comment
+ *         postId:
+ *           type: string
+ *           description: The id of the post that the comment belongs to
+ *         userId:
```
Comment (danielpasker) [inline src/routes/commentsRoutes.ts:31]: UserId is not required in all the routes. We decode it from the given jwt

**Review by danielpasker — CHANGES_REQUESTED** [review]
Done @shaynisanov

File: src/routes/commentsRoutes.ts (hunk 2 above)
Comment (shaynisanov) [reply src/routes/commentsRoutes.ts:31]: removed

File: src/server.ts (hunk 1 above)
Comment (shaynisanov) [reply src/server.ts]: done

**Pull request merged**